README.md

```diff
@@ -4,6 +4,8 @@
 
 DIGITS (the **D**eep Learning **G**PU **T**raining **S**ystem) is a webapp for training deep learning models.
 
+The currently supported frameworks are: Caffe 1, Torch, and Tensorflow
+
```

Comment:
Remove the newline so this section is just one paragraph. Call it just "Caffe", not "Caffe 1". Add a period at the end.